### PR TITLE
refactor: change ampd finalizer variant names

### PR DESCRIPTION
**Pull request by maancham**
## Description

AXE-3387

## Todos

- [ ] Unit tests
- [ ] Manual tests
- [ ] Documentation
- [ ] Connect epics/issues

### CODE REVIEW

File: ampd/src/evm/finalizer.rs [hunk 1]
```diff
@@ -70,31 +70,31 @@ where
     }
 }
 
-pub struct PoWFinalizer<'a, C>
+pub struct ConfirmationFinalizer<'a, C>
```
Comment (cjcobb23) [inline ampd/src/evm/finalizer.rs:73]:
```suggestion
pub struct ConfirmationHeightFinalizer<'a, C>
```

Comment (maancham) [reply ampd/src/evm/finalizer.rs:73]: Done

File: ampd/src/evm/finalizer.rs [hunk 2]
```diff
@@ -32,31 +32,32 @@ where
...
+        Finalization::RPCFinalizedBlock => Box::new(RPCBlockFinalizer::new(rpc_client)),
+        Finalization::ConfirmationHeight => Box::new(ConfirmationHeightFinalizer::new(
+            rpc_client,
+            confirmation_height,
+        )),
     }
 }
 
-pub struct EthereumFinalizer<'a, C>
+pub struct RPCBlockFinalizer<'a, C>
```
Comment (cgorenflo) [inline ampd/src/evm/finalizer.rs:43]:
```suggestion
pub struct RPCFinalizer<'a, C>
```

**Review by cgorenflo — APPROVED** [review]

File: ampd/src/evm/finalizer.rs (hunk 2 above)
Comment (maancham) [reply ampd/src/evm/finalizer.rs:43]: Done.

**Pull request merged**